src/tests/test_analysis_intellij.py

```diff
@@ -16,6 +16,9 @@ def test_analysis_intellij(configurations, setup_intellij, app_name, analysis_da
     intellij = setup_intellij
     application_data = analysis_data[app_name]
     expected_story_points = application_data["story_points"]
+    configurations_object, html_file_location = configurations
+    intellij.configurations = configurations_object
```

Comment:
Hi @midays , thanks for the PR.

I'd like to point out a few things.
1) We already have this on line 27 -
_, html_file_location = configurations
2)'intellij.configurations' is not being used anywhere in the file.